opencti-platform/opencti-front/src/private/components/threats/ThreatActorsGroup.jsx

```diff
@@ -133,15 +133,16 @@ class ThreatActorGroups extends Component {
         paginationOptions={paginationOptions}
         numberOfElements={numberOfElements}
         availableFilterKeys={[
+          'x_opencti_workflow_id',
           'labelledBy',
           'markedBy',
-          'created_start_date',
-          'created_end_date',
           'createdBy',
+          'source_reliability',
```

Comment:
This filter seems to not working properly
![image](https://github.com/OpenCTI-Platform/opencti/assets/53513584/a9132e77-a92e-440c-817e-a129b86df473)

![image](https://github.com/OpenCTI-Platform/opencti/assets/53513584/d788262a-bea0-47c3-b391-fe9467fd4682)

![image](https://github.com/OpenCTI-Platform/opencti/assets/53513584/13c0ce43-aed6-41e9-b90a-a6b4c8817819)

Reply:
You're right, I didn't see it, it was because the filter didn't match any author, so all results were displayed. It should be fixed now @RomuDeuxfois